packages/html/src/templates/html.css

```diff
@@ -0,0 +1,179 @@
+:root {
+  --color-dark: #252525;
+  --color-light: #efefef;
+  --color-primary: #1a8fe3;
+  --size-step-0: clamp(1rem, calc(0.96rem + 0.22vw), 1.13rem);
+  --size-step-1: clamp(1.25rem, calc(1.16rem + 0.43vw), 1.5rem);
+  --size-step-2: clamp(1.56rem, calc(1.41rem + 0.76vw), 2rem);
+  --size-step-3: clamp(1.95rem, calc(1.71rem + 1.24vw), 2.66rem);
+  --size-step-4: clamp(2.44rem, calc(2.05rem + 1.93vw), 3.55rem);
+}
+
+h1,
+h2,
+h3 {
+  font-family: Inter, sans-serif;
```

Comment:
Users will likely not have Inter installed by default. Either we can opt for a list of more standard fonts or import Inter from [Google fonts](https://fonts.google.com/specimen/Inter?query=inter).

Importing can be done either using a `<link>` tag in the HTML doc or `@import` in your css.
Something like this (note that we likely need to include more font weights then this example) :
```
@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap');
```